[fix] Fix error of showing more than one reply form on comment dialog

joassouza opened the pull request:
This fix the error of showing more than one reply form per comment
dialog

joassouza committed:
[fix] Remove all the forms when recreate dialog

This fix the error of showing more than one replies form per comment
dialog

lpagliari commented on static/js/commentInfoDialog.js:
```diff
@@ -389,8 +389,7 @@ commentInfoDialog.prototype._addDateFieldToComment = function($infoDialog, comme
 commentInfoDialog.prototype._addReplyCommentField = function($infoDialog, commentData) {
-  var addReplyFormId = this._getAddReplyFormId(commentData.commentId);
-  $infoDialog.find(addReplyFormId).remove(); // remove previous forms
+  $infoDialog.find('form.new-reply').remove(); // remove previous forms
```
se a gente não pode ter 2 forms no mesmo dialog, faz sentido criar o form usando o `commentId` (ver `_getAddReplyFormId`)? Não seria mais simples ter apenas um form genérico?

joassouza replied:
faz muito sentido o que vc falou. Ter um id único é um erro de design 👍

joassouza committed:
[refactor] Address code review suggestions

We don't need an unique id for reply field since we only can remove one
field per comment dialog
